### ISSUE TITLE
Review default configuration for emacs-metals

### ISSUE THREAD

**Issue by kpbochenek** [emacs-lsp/lsp-metals#5]
Hi,

we have a default configuration on our site for emacs (https://scalameta.org/metals/docs/editors/emacs.html)

Today issue was raised that it is no longer valid because now you have to additionally require `lsp-metals` in addition to `lsp-mode`. As I understand it is related to moving to separate repo?

I also noticed that `lsp-metals-treeview-enable` is no longer valid. Can this be safely removed?

Could you just take a look if anything else should be adjusted?

**Comment by kurnevsky** [comment]
Yes, we moved it to a separate repo to resolve dependency issues, `lsp-metals` depends on both `lsp-mode` and `dap-mode` and includes treeview now.

> I also noticed that lsp-metals-treeview-enable is no longer valid. Can this be safely removed?

Yes, before we had to explicitly enable it because we couldn't depend on `lsp-treemacs`. Now it's always enabled.

> Could you just take a look if anything else should be adjusted?

`(setq lsp-metals-treeview-show-when-views-received t)` should be moved to `lsp-metals` config, everything else seems valid. I can make a PR later.
